main.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+var user, guess, answer, hint1, message;
+
+var MyGuessingGame = function() {
+  	
```

Comment:
Way too much whitespace here. You can clean this up and keep it readable without all the excess whitespace.